README.rst

```diff
@@ -7,6 +7,10 @@ Single Cell Tools
 .. image:: https://codecov.io/gh/HumanCellAtlas/sctools/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/HumanCellAtlas/sctools
 
+.. image:: https://readthedocs.org/projects/sctools/badge/?version=latest
```

Comment:
nit: By clicking the rich diff, the badge is not aligned with other existing badges, not sure if we have to remove the above blank line?

Reply:
Seems aligned to me. Am I not understanding?

![image](https://user-images.githubusercontent.com/5239152/46879922-46aa4880-ce15-11e8-89b9-56712747a08a.png)